### ISSUE TITLE
Tensorflow task example

### ISSUE THREAD

**Issue by stirby** [coder/internal#811]
Want to create a task to train a simple model.

The model should train on some visual dataset like MNIST digits so there's a fun `coder_app` output at the end. 

A GPU should be attached. 

Parameters should define training process
- Learning Rate
- Optimizer
- Epochs
